library/include/hipblas.h

```diff
@@ -7103,6 +7149,264 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasTrsmStridedBatchedEx(hipblasHandle_t    ha
+//                                               const void*        A,
+//                                               hipblasDatatype_t  Atype,
+//                                               int                lda,
+//                                               const float*       beta,
+//                                               hipblasComplex*    C,
+//                                               hipblasDatatype_t  Ctype,
+//                                               int                ldc);
+
+// axpy_ex
+HIPBLAS_EXPORT hipblasStatus_t hipblasAxpyEx(hipblasHandle_t   handle,
+                                             int               n,
+                                             const void*       alpha,
+                                             hipblasDatatype_t alphaType,
+                                             const void*       x,
+                                             hipblasDatatype_t xType,
+                                             int               incx,
+                                             void*             y,
+                                             hipblasDatatype_t yType,
+                                             int               incy,
+                                             hipblasDatatype_t executionType);
+
+HIPBLAS_EXPORT hipblasStatus_t hipblasAxpyBatchedEx(hipblasHandle_t   handle,
+                                                    int               n,
+                                                    const void*       alpha,
+                                                    hipblasDatatype_t alphaType,
+                                                    const void*       x,
+                                                    hipblasDatatype_t xType,
+                                                    int               incx,
+                                                    void*             y,
+                                                    hipblasDatatype_t yType,
+                                                    int               incy,
+                                                    int               batch_count,
+                                                    hipblasDatatype_t executionType);
+
+HIPBLAS_EXPORT hipblasStatus_t hipblasAxpyStridedBatchedEx(hipblasHandle_t   handle,
+                                                           int               n,
+                                                           const void*       alpha,
+                                                           hipblasDatatype_t alphaType,
+                                                           const void*       x,
+                                                           hipblasDatatype_t xType,
+                                                           int               incx,
+                                                           int               stridex,
```

Comment:
Aren't these strides to all be int64, some cuda seem to be long long int .... ?

Reply:
Yeah, strides are currently `int` across hipBLAS and need to be updated to at least match rocBLAS, I'll make a ticket.